### DISCUSSION TITLE
Introducing the Cachet Core

### DISCUSSION THREAD

**Discussion by jbrooksuk** [cachethq/cachet#4348] [Announcements]
Hi all,

[We've just open-sourced the `cachethq/core`](https://blog.cachethq.io/cachet-core/) package. As the name suggests. this is now home to the core of Cachet. This includes the status page and dashboard elements.

This is still a very, very early release of the core package and it's absolutely not ready for use yet. However, this does give us all a place to work from.

Of course, we're open to suggestions and feedback on this project.

Once this project has developed further, `cachethq/cachet` will have a new 3.x branch that will require this package.

I look forward to developing the core of Cachet 3.x further with you all 👋🏻 

— James

**Comment by ricardov03** [comment]
Hello @jbrooksuk. What do you expect from us as developers at this stage? I would like to know if you want us to review the core and make any suggestions based on what we see. Or do you prefer we wait until you start the Roadmap?

**Reply by jbrooksuk** [reply]
Sorry, that wasn't clear at all from my initial post. 

From the community, I would love feedback on the project so far, contributions to the code that I've stubbed out and areas I've missed. 

Ideally, the smaller the pull request, the better.
